lib/Renard/Curie/Component/PageDrawingArea.pm

```diff
@@ -227,6 +227,9 @@ method setup_drawing_area() {
 
 	my $vbox = $self->builder->get_object('page-drawing-component');
 	$vbox->pack_start( $scrolled_window, TRUE, TRUE, 0);
+
+	$scrolled_window->add_events([ 'scroll-mask', 'smooth-scroll-mask' ]);
```

Comment:
This line is not needed. It already supports scrolling.

TODO: check this statement.